README_zh.md

```diff
@@ -161,6 +161,7 @@ https://github.com/hiyouga/LLaMA-Factory/assets/16256802/ec36a9dd-37f4-4f72-81bd
 | [Falcon](https://huggingface.co/tiiuae)                      | 7B/11B/40B/180B                  | falcon    |
 | [Gemma/Gemma 2/CodeGemma](https://huggingface.co/google)     | 2B/7B/9B/27B                     | gemma     |
 | [GLM-4](https://huggingface.co/THUDM)                        | 9B                               | glm4      |
+| [GLM-4V](https://huggingface.co/THUDM)                       | 13B(include vision)              | glm4_v    |
```

Comment:
No need for add vision's size to 13B (Don't add expression like "include vision")
Just keep consistent with llava/paligemma/yivl, using 9B as official report

Reply:
Ok. i got it.(my firend wrote it)